easybuild/easyblocks/b/bowtie2.py

```diff
@@ -16,12 +16,13 @@
 @author: Fotis Georgatos (Uni.Lu)
 @author: Kenneth Hoste (Ghent University)
 """
-
+from distutils.version import LooseVersion
 import os
 import shutil
 
 from easybuild.easyblocks.generic.configuremake import ConfigureMake
 from easybuild.tools.build_log import EasyBuildError
+from easybuild.tools.filetools import copy_file
 
 
 class EB_Bowtie2(ConfigureMake):
```

Comment:
because then I have to fiddle with the mandatory `files_to_copy` easyconfig parameter, not worth the trouble here imho